cloudslang-runtime/src/main/resources/scripts/cs_extract_number.py

```diff
@@ -0,0 +1,12 @@
+def cs_extract_number(str, count = 1):
+  nums = str.split()
```

Comment:
Same as the above. If we expect string here, we should enforce that with a check. Or else, `str.split()` will fail